src/core_plugins/kibana/public/home/components/home.js

```diff
@@ -49,173 +39,31 @@ export function Home({ addBasePath, directories, isCloudEnabled }) {
-        <KuiCardGroup>
-
-          {apmCard}
-
-          <KuiCard style={cardStyle} className="euiPanel">
-            <KuiCardDescription>
-              <KuiCardDescriptionTitle>
-                <img
-                  src={addBasePath('/plugins/kibana/assets/app_logging.svg')}
-                />
-                <p>
-                  Logging
-                </p>
-              </KuiCardDescriptionTitle>
-
-              <KuiCardDescriptionText>
-                Ingest logs from popular data sources and easily visualize in preconfigured dashboards.
-              </KuiCardDescriptionText>
-            </KuiCardDescription>
-
-            <KuiCardFooter>
-              <KuiLinkButton
-                buttonType="secondary"
-                href={addBasePath(`${kbnBaseUrl}#/home/tutorial_directory/logging`)}
-              >
-                Add log data
-              </KuiLinkButton>
-            </KuiCardFooter>
-          </KuiCard>
-
-          <KuiCard style={cardStyle} className="euiPanel">
-            <KuiCardDescription>
-              <KuiCardDescriptionTitle>
-                <img
-                  src={addBasePath('/plugins/kibana/assets/app_monitoring.svg')}
-                />
-                <p>
-                  Metrics
-                </p>
-              </KuiCardDescriptionTitle>
-
-              <KuiCardDescriptionText>
-                Collect metrics from the operating system and services running on your servers.
-              </KuiCardDescriptionText>
-            </KuiCardDescription>
-
-            <KuiCardFooter>
-              <KuiLinkButton
-                buttonType="secondary"
-                href={addBasePath(`${kbnBaseUrl}#/home/tutorial_directory/metrics`)}
-              >
-                Add metric data
-              </KuiLinkButton>
-            </KuiCardFooter>
-          </KuiCard>
-
-          <KuiCard style={cardStyle} className="euiPanel">
-            <KuiCardDescription>
-              <KuiCardDescriptionTitle>
-                <img
-                  src={addBasePath('/plugins/kibana/assets/app_security.svg')}
-                />
-                <p>
-                  Security analytics
-                </p>
-              </KuiCardDescriptionTitle>
-
-              <KuiCardDescriptionText>
-                Centralize security events for interactive investigation in ready-to-go visualizations.
-              </KuiCardDescriptionText>
-            </KuiCardDescription>
-
-            <KuiCardFooter>
-              <KuiLinkButton
-                buttonType="secondary"
-                href={addBasePath(`${kbnBaseUrl}#/home/tutorial_directory/security`)}
-              >
-                Add security events
-              </KuiLinkButton>
-            </KuiCardFooter>
-          </KuiCard>
-        </KuiCardGroup>
-      </div>
+  let recentlyAccessedPanel;
+  if (recentlyAccessed.length > 0) {
+    recentlyAccessedPanel = (
+      <React.Fragment>
```

Comment:
In general, I think we're importing submodules instead of referring to them as properties on the main module. I think this is more "ES6-y"?

```jsx
import React, { Fragment } from 'react';

<Fragment />
```